src/main/features/response/views/eligibility/send-your-response-by-email.njk

```diff
@@ -0,0 +1,99 @@
+{% extends "layout.njk" %}
+{% from "support.njk" import supportWidget %}
+{% from "form.njk" import csrfProtection, submitButton %}
+
+{% set title = t('Fees | Money claim') %}
+{% set heading = t('Send your response by email') %}
+
+{% macro newWindow() %}<span class="font-xsmall">{{ t('(opens in a new window)') }}</span>{% endmacro %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <div class="panel-indent">
+        <div class="panel panel-border-wide">
+          <p>{{ t('You must email your response by ') }}
+            <span class="bold">{{ t('%s', { postProcess: 'sprintf', sprintf: [claim.responseDeadline | date]}) }}</span>
+          </p>
+        </div>
+      </div>
+
+      {% if draft.response.type === ResponseType.DEFENCE %}
+        {% if draft.rejectAllOfClaim.option === RejectAllOfClaimOption.COUNTER_CLAIM %}
+
+        <h2 class="heading-medium">{{ t('Counterclaim fee') }}</h2>
+        <p>
+          {{ t('You`ll need to pay a court fee if you make a counterclaim.')}}
+        </p>
+        <details class="notice">
+          <summary>{{ t('How these fees are calculated') }}</summary>
+
+          <div class="panel">
+            <p>
+              {{ t('The fee is based on the amount you`re claiming, including interest.') }}
+            </p>
+            <table>
+              <thead>
+              <tr>
+                <th scope="col">Claim amount</th>
+                <th scope="col">Fee</th>
+              </tr>
+              </thead>
+              {% for row in rows %}
+                <tr>
+                  <td>{{ row.from | pennies2pounds | numeral }} to {{ row.to | pennies2pounds | numeral }}</td>
+                  <td>{{ row.fees[1] | pennies2pounds | numeral }}</td>
```

Comment:
Why you make two calls to retrieve both issue and hearing fee while in the frontend you just use one? Please rework it to fetch only what is needed.